keadm/README.md

```diff
@@ -1,17 +1,263 @@
-# KubeEdge Installer
+# Getting Started with KubeEdge Installer
 
-[Kubeedge Installer Doc](<https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/kubeedgeadm-scope.md/>)
+Please refer to KubeEdge Installer proposal document for details on the motivation of having KubeEdge Installer.
```

Comment:
@srivatsav123 , the design doc has seperate commands and implementation is different. Please update the design doc.

Reply:
ok